pkgs/development/compilers/scala-runners/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, jre, coursier }:
+
+let 
+  name = "scala-runners";
+  owner = "dwijnand";
+in stdenv.mkDerivation {
+  inherit name;
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/${owner}/${name}";
+    description = "An alternative implementation of the Scala distribution's runners";
+    license = licenses.asl20;
+    platforms = platforms.all;
+    maintainers = [ maintainers.hrhino ];
+  };
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = name;
+    rev = "0c0b3690b4126d6dfd15024fc759da07434e7b51";
+    sha256 = "1cqb96psf54dmwzq9fvbr4nwgakmq3dv26c09m4rp8ikakpid3pd";
+  };
+
+  installPhase = ''
+    set -x
+    mkdir -p $out/bin $out/lib
+    sed -ie "s_\Wcs\W_ ${coursier}/bin/coursier _" scala-runner
+    cp scala-runner $out/lib
+    ln -s $out/lib/scala-runner $out/bin/scala
+    ln -s $out/lib/scala-runner $out/bin/scalac
+    ln -s $out/lib/scala-runner $out/bin/scalap
+    ln -s $out/lib/scala-runner $out/bin/scaladoc
+  '';
+
+
```

Comment:
```suggestion
  installPhase = ''
    mkdir -p $out/bin $out/lib
    sed -ie "s_\Wcs\W_ ${coursier}/bin/coursier _" scala-runner
    cp scala-runner $out/lib
    ln -s $out/lib/scala-runner $out/bin/scala
    ln -s $out/lib/scala-runner $out/bin/scalac
    ln -s $out/lib/scala-runner $out/bin/scalap
    ln -s $out/lib/scala-runner $out/bin/scaladoc
  '';
```

Reply:
what changed here? just removing the `set -x`? I guess that's a good idea.

Reply:
yeah. not sure why I selected more lines.